src/app/app.component.less

```diff
@@ -1,3 +1,7 @@
 .f8-wi-container {
   margin-top: 34px; /* workaround for planner tab in fabric8-ui layout */
 }
+
+.f8-modal-lg {
+  width: 70%;
```

Comment:
Please change this to:
```
width: 1500px;
max-width: calc(100% - 20px);
```
This way the size stays responsive.
The `-20px` accounts for the modal margins.
`1500px` is somewhat arbitrary as it looks good for large desktop screens and scales accordingly with the above css change.